src/citrine/informatics/predictors/mean_property_predictor.py

```diff
@@ -0,0 +1,105 @@
+from typing import List, Optional, Mapping
+
+from citrine._serialization import properties as _properties
+from citrine._serialization.serializable import Serializable
+from citrine._session import Session
+from citrine.informatics.data_sources import DataSource
+from citrine.informatics.descriptors import FormulationDescriptor, RealDescriptor
+from citrine.informatics.reports import Report
+from citrine.informatics.predictors import Predictor
+
+__all__ = ['MeanPropertyPredictor']
+
+
+class MeanPropertyPredictor(
+        Serializable['MeanPropertyPredictor'], Predictor):
+    """[ALPHA] A predictor interface that computes mean component properties.
+
+    .. seealso::
+       If you are using a deprecated generalized mean property predictor please see
+       :class:`~citrine.informatics.predictors.GeneralizedMeanPropertyPredictor`
+       for details on how to migrate to the new format.
```

Comment:
I don't see these details in `GeneralizedMeanPropertyPredictor`

Reply:
I don't know where those changes went. Hold on while I rewrite it...

Reply:
Just pushed the docs.